test/lib/token.test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const assert = require('chai').assert;
+const sinon = require('sinon');
+const mockery = require('mockery');
+const schema = require('screwdriver-data-schema');
+
+sinon.assert.expose(assert, { prefix: '' });
+require('sinon-as-promised');
+
+describe('Token Model', () => {
+    const password = 'password';
+    let BaseModel;
+    let TokenModel;
+    let datastore;
+    let createConfig;
+    let token;
+
+    before(() => {
+        mockery.enable({
+            useCleanCache: true,
+            warnOnUnregistered: false
+        });
+        datastore = {
+            update: sinon.stub()
+        };
+    });
+
+    beforeEach(() => {
+        datastore.update.resolves({});
+
+        /* eslint-disable global-require */
+        BaseModel = require('../../lib/base');
+        TokenModel = require('../../lib/token');
+        /* eslint-enable global-require */
+
+        createConfig = {
+            datastore,
+            userId: 12345,
+            uuid: '1a2b3c',
+            id: 6789,
+            name: 'Mobile client auth token',
+            description: 'For the mobile app',
+            lastUsed: '2017-05-10T01:49:59.327Z',
+            password
+        };
+        token = new TokenModel(createConfig);
+    });
+
+    after(() => {
+        mockery.deregisterAll();
+        mockery.resetCache();
+        mockery.disable();
```

Comment:
If you're doing a `.disable()`, you don't need `.deregisterAll()` or `.resetCache()`